test-resources/headless-core-stub/lambdas/mock-jwks/src/mock-jwk-handler.ts

```diff
@@ -19,7 +19,7 @@ export class MockJwkHandler implements LambdaInterface {
                     statusCode: 200,
                     body: JSON.stringify(jwks),
                     headers: {
-                        "Cache-Control": "max-age=900",
+                        "Cache-Control": "max-age=300",
```

Comment:
Why'd we change this? 👀

Reply:
I found https://github.com/govuk-one-login/ipv-core-back/blob/6dcd1644c0bfd1dcfc41c3692d73892933ed82e5/openAPI/core-back-external.yaml#L162